sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/LocalRelation.scala

```diff
@@ -62,6 +62,10 @@ case class LocalRelation(output: Seq[Attribute], data: Seq[InternalRow] = Nil)
     case _ => false
   }
 
+  override def simpleString: String =
+    if (data == Seq.empty) super.simpleString + " [Empty Seq]"
```

Comment:
should be https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/trees/TreeNode.scala#L401